### PR TITLE
Update document list design to match design

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Update product document list to match design.

## Justify why they are needed

Left out the description texts for now.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1574** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1574" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1574?utm_source=stack-comment).

File: packages/ui/src/icons/NeArrow.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { IconRoot, IconRootProps } from './Root'
+
+export const NeArrow = ({ size = '1.5rem', color = 'currentColor', ...props }: IconRootProps) => {
```
Comment (gustaveen) [inline packages/ui/src/icons/NeArrow.tsx:3]: Ne as in north east? 🤓

**Review by gustaveen — APPROVED** [review]
Nice! 👌

File: packages/ui/src/icons/NeArrow.tsx (hunk 1 above)
Comment (robinandeer) [reply packages/ui/src/icons/NeArrow.tsx:3]: hehe I guess so, just copied from figma

**Comment by robinandeer** [comment]
⏳ This pull request is set to merge as part of a Graphite merge job
Stack job ID: 13hOaMCQytMdTvkr5Ynk.
See details on [graphite.dev](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1574)

**Pull request merged**